doc/latest_news.html

```diff
@@ -4,6 +4,7 @@
       December 22nd 2015: HoloViews 1.4.1 released and now available on 
       <a href="https://pypi.python.org/pypi/holoviews/">PyPI</a> and
       <a href="https://anaconda.org/anaconda/holoviews">Anaconda</a>.
+      Now includes full support for the <a href="http://bokeh.pydata.org">Bokeh</a> plotting library.
```

Comment:
Well the Plotly support has not been merged so no one should be under any illusion that it has full support. I did try to raise the issue of what it means for a backend to be ready for inclusion in HoloViews in that PR. As you point out in future there will likely not be one single backend that supports all the Elements. If we ever have cross-backend layout capabilities this won't be such an issue but for now people will have to very deliberately use backends for particular purposes. I think that's reasonable as long as it's easy for people to find out about the drawbacks and benefits of particular backends and what is supported. I don't think we should be overly restrictive about what constitutes a supported backend, since some backends might only support a few Element types but draw them very quickly.